flamedisx/x1t_sr1.py

```diff
@@ -29,12 +29,23 @@
 
 
 @export
-class SR1Source(fd.ERSource):
+class SR1Source:
     drift_velocity = 1.335 * 1e-4   # cm/ns
     extra_needed_columns = tuple(
         list(fd.ERSource.extra_needed_columns)
         + ['x_observed', 'y_observed'])
 
+    def random_truth(self, energies, fix_truth=None, **params):
+        d = super().random_truth(energies, fix_truth=fix_truth, **params)
```

Comment:
`energies` was renamed to `n_events` in #30 since we don't support arrays here any more. I'd rename it for consistency.